hubs/my-train-company/trainbook-openapi-source.yaml

```diff
@@ -0,0 +1,1112 @@
+openapi: 3.1.0
+info:
+  title: Train Travel API
+  description: |
+    API for finding and booking train trips across Europe.
+
+    This API was created by [Bump.sh](https://bump.sh/) for the entire OpenAPI
+    community, for educational and demonstrative purposes.
+  version: 1.0.0
+  contact:
+    name: Train Support
+    url: https://example.com/support
+    email: support@example.com
+  license: 
+    name: Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International
+    identifier: CC-BY-NC-SA-4.0
+
+servers:
+  - url: https://api.example.com
+    description: Production
+
+security:
+  - OAuth2:
+      - read
+
+x-topics:
+  - title: Getting started
+    content:
+      $ref: ./docs/getting-started.md
```

Comment:
I think your missing an external ref file. Could you also add that file in the PR (to make sure the deploy works fine).